### PR TITLE
Login User Interface

### PR DESCRIPTION
**Pull request by shiara02**
Create login user interface
image

### CODE REVIEW

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

File: src/routes/web.php
Comment (nadj1233) [inline src/routes/web.php:1]: Dont expose your logic here, let the controller handle it.

File: src/resources/views/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+</body>
+</html>
```
Comment (hipe01mark) [inline src/resources/views/login.blade.php]: Use laravel blade templates to layout your view files.

File: src/routes/web.php [hunk 2]
```diff
@@ -14,5 +14,5 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('login');
 });
```
Comment (hipe01mark) [inline src/routes/web.php]: Create a separate file for the controller.

File: src/app/Http/Controllers/LoginController.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class LoginController extends Controller
+{
+    public function index()
+    {
+        return view('auth/login');
```
Comment (hipe-francis) [inline src/app/Http/Controllers/LoginController.php:11]: is this working? shouldn't the syntax be `view('path.file')`?

File: src/resources/views/login.blade.php (hunk 1 above)
Comment (shiara02) [reply src/resources/views/login.blade.php]: I have resolved this comment, sir. Thank you!

File: src/routes/web.php (hunk 2 above)
Comment (shiara02) [reply src/routes/web.php]: I have resolved this comment, sir. Thank you!

File: src/app/Http/Controllers/LoginController.php (hunk 3 above)
Comment (shiara02) [reply src/app/Http/Controllers/LoginController.php:11]: Yes, sir. It's working po. Ayan po kasi yung nakita kong ginagamit sa routing. Should I edit it po, sir? Thank you po.

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

File: src/resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
+@extends('layout/layout')
+
+<body>
+    <section class="vh-100" style="background-color: #023047;">
```
Comment (hipe-francis) [inline src/resources/views/auth/login.blade.php]: Fix this once ma merge yung UI for this

File: src/resources/views/layout/layout.blade.php [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.min.js"></script>
+
+</head>
```
Comment (hipe01mark) [inline src/resources/views/layout/layout.blade.php]: Already have a layout folder with a layout file again? Is this still in WIP with Karl?

File: src/routes/web.php [hunk 6]
```diff
@@ -13,6 +14,4 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', [LoginController::class, 'index']);
```
Comment (hipe01mark) [inline src/routes/web.php]: add new line

File: src/resources/views/layout/layout.blade.php (hunk 5 above)
Comment (shiara02) [reply src/resources/views/layout/layout.blade.php]: Yes po sir, sa end po ni Karl maayos na po yung files under layout folder. Will fix po after merging baka po kasi magka conflict. Thank you po.

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: src/routes/web.php (hunk 6 above)
Comment (bpocrafael) [inline src/routes/web.php]: trailing white space?

File: src/app/Http/Controllers/LoginController.php [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class LoginController extends Controller
+{
+    public function index()
```
Comment (bpocrafael) [inline src/app/Http/Controllers/LoginController.php]: return type

**Review by bpocrafael — DISMISSED** [review]
One last comment but LGTM 🚢

File: src/app/Http/Controllers/LoginController.php (hunk 7 above)
Comment (shiara02) [reply src/app/Http/Controllers/LoginController.php]: I already resolved this comment, sir. Thank you po!

**Review by nadj1233 — APPROVED** [review]
LGTM 🐐

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]

**Pull request merged**